pkgs/applications/science/logic/aspino/default.nix

```diff
@@ -1,34 +1,48 @@
-{ stdenv, fetchFromGitHub, zlib, boost, glucose }:
+{ stdenv, fetchurl, fetchFromGitHub, zlib, boost, glucose }:
+
+let
+  glucose' = glucose.overrideAttrs (_: {
+    src = fetchurl {
+      url = "http://www.labri.fr/perso/lsimon/downloads/softwares/glucose-syrup.tgz";
```

Comment:
I copied this URL from the last `glucose` bump, however a versioned URL is preferable

Reply:
unfortunately the Glucose homepage currently points to this URL as well for v4.0 (http://www.labri.fr/perso/lsimon/glucose/), furthermore `http://www.labri.fr/perso/lsimon/downloads/softwares/glucose-syrup-4.0.tgz` returns a `404` so I'm afraid that we have to keep this for now